lib/go/test/flow_epoch_test.go

```diff
@@ -882,15 +894,15 @@ func TestEpochQCDKG(t *testing.T) {
 				startView:             startView + numEpochViews,
 				endView:               startView + 2*numEpochViews - 1,
 				stakingEndView:        startView + numEpochViews + numStakingViews - 1,
-				totalRewards:          "1300000.0",
+				totalRewards:          "6572143.3875",
```

Comment:
We should also check here that the rewards are paid out in the `IDTableStaking` contract, for example ensuring the tokens are moved into the rewards bucket.